### ISSUE TITLE
Custom dynamic multiple selector row.

### ISSUE THREAD

**Issue by morugu** [xmartlabs/XLForm#805]
I want to display all selected values in row.
If user selected many values, row's height increase from default size to than big.
like that image. 
https://drive.google.com/file/d/0B9VYuGIqVQihSnV1VFJSR2huNVU/view?usp=sharing

What would be the best thing to do?

**Comment by mats-claassen** [comment]
Selector cells use the default textLabel and detailTextLabel so you have to play with them to get your desired cell.

This two lines will be a good start for you:
```
[row.cellConfig setValue:@0 forKey:@"detailTextLabel.numberOfLines"];
row.height = UITableViewAutomaticDimension;
```

**Comment by morugu** [comment]
@mats-claassen 
is there `height` property in XLFormRowDescriptor ?

**Comment by morugu** [comment]
I tried set config and become like that.
https://drive.google.com/file/d/0B9VYuGIqVQihMmZRT0N6RlRvOG8/view?usp=sharing

I wanna fix it.

**Comment by mats-claassen** [comment]
The `height` property is in XLForm since the latest 3.2.0 release. It had been in master branch for a while.

**Comment by morugu** [comment]
thanks for release 3.2.0. I confirmed adding `height` property.
however, not yet fixed this bug.
https://drive.google.com/file/d/0B9VYuGIqVQihMmZRT0N6RlRvOG8/view?usp=sharing

**Comment by mats-claassen** [comment]
Ok I am seeing this now as well.
I believe I had this working but I discarded the code I was playing around with and it seems that there was something more I did to make this work which I do not remember.

Anyway [here](http://stackoverflow.com/questions/36587126/autolayout-ignores-multi-line-detailtextlabel-when-calculating-uitableviewcell-h) is a stack overflow question about pretty much the same thing.

Let me know if you fix it.
